packages/genesys-spark-components/src/components/stable/gux-tag/gux-tag.scss

```diff
@@ -32,15 +32,8 @@
 
   &.gux-disabled {
     position: relative;
-
-    &::before {
-      position: absolute;
-      inset: 0;
-      content: '';
-      background-color: #fff;
-      border-radius: var(--gse-ui-tag-borderRadius);
-      opacity: var(--gse-ui-tag-disabled-opacity);
-    }
+    user-select: none;
+    opacity: var(--gse-ui-tag-disabled-opacity);
   }
```

Comment:
Dont think the `::before` was doing anything so just removed it and added the opacity to the gux-disabled class so that the background-color is no longer needed.